prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -169,6 +175,8 @@ postsubmits:
     annotations:
       testgrid-dashboards: googleoss-test-infra
       testgrid-tab-name: hmacs-reconciliation
+      testgrid-alert-email: k8s-infra-oncall@google.com
+      testgrid-num-failures-to-alert: '2'
```

Comment:
nit: We might want to know when the hmac reconciler fails since that would probably need to be rerun even for a single transient failure. Otherwise there would be a missing webhook or a secret that was not rotated and it wouldn't be addressed until the next change to the webhook config which could be a long while.

Reply:
Done